changes/ce/feat-13180.en.md

```diff
@@ -0,0 +1 @@
+Improve client message handling performance when running on OTP 26.
```

Comment:
So we removed the function of conn congestion alarm?

Reply:
it's still called, but only triggered by the stats emit timer.